modules/ROOT/partials/configuration/ui_mode.adoc

```diff
@@ -0,0 +1,41 @@
+== Editor in scrollable container
+
+[[ui_mode]]
+=== `+ui_mode+`
+
+include::partial$misc/admon-requires-6.4v.adoc[]
+
+This option configures the location of the container where popup menus and inline dialogs are rendered. 
+
+By default, these elements are rendered in a container located in the body.
+
+When the editor is rendered in a scrollable container, scrolling the editor’s container will cause popup menus and inline dialogs to be detached from the anchor.
+
+If `+toolbar_sticky+` is set to true, the sticky toolbar and the menu have no effect on docking when scrolling the editor’s container; they are only triggered when the page is scrolled.
+
+However, setting this option to `+'split'+` enables support for editors in scrollable containers and adjusts the behaviour as follows:
+
+* Popup menus and inline dialogs are rendered in a separate container adjacent to the editor. These elements moves together as you scroll the editor’s container. 
```

Comment:
```suggestion
* Popup menus and inline dialogs are rendered in a separate container adjacent to the editor. These elements move together as you scroll the editor’s container. 
```

Reply:
They are already rendered on a separate container (a different sink). I believe, when the configuration is set to `split`,  these two will be put on separated sinks.